lib/rules/strict.js

```diff
@@ -17,7 +17,9 @@ var messages = {
     global: "Use the global form of \"use strict\".",
     multiple: "Multiple \"use strict\" directives.",
     never: "Strict mode is not permitted.",
-    unnecessary: "Unnecessary \"use strict\" directive."
+    unnecessary: "Unnecessary \"use strict\" directive.",
+    unnecessaryInModules: "Unnecessary \"use strict\" directive, because inside of modules.",
```

Comment:
I think the more info we can give people, the better a tool we are.

@mysticatea I would reword these slightly:

1. "use strict" is unnecessary inside of modules
1. "use strict" is unnecessary inside of classes